### PR TITLE
Yessenia Rodriguez Javascript typing tutor

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-02-20 at 7 22 57 PM

### CODE REVIEW

File: javascript-typing-tutor/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+firstSpan.className = 'cursor';
+
+function matchKeys ({ key }) {
+  for (var j = 0; j < span.length; j++) {
+    var currentCharacter = text[j];
+    if (key === currentCharacter) {
+      span[j].removeAttribute('class', 'cursor');
+      span[j].className = 'correct';
+      span[++j].className = 'cursor';
+    }
```
Comment (thebearingedge) [inline javascript-typing-tutor/main.js:27]: What if they get it incorrect, does the user know? There should be a visual indication.

File: javascript-typing-tutor/main.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+var text = 'hi world';
+var $tutor = document.querySelector('.tutor');
+var characters = text.split('');
+
+
+function createSpan() {
```
Comment (thebearingedge) [inline javascript-typing-tutor/main.js:6]: This creates more than one span. Be careful with function names!

File: javascript-typing-tutor/main.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  }
+}
+
+createSpan();
+var span = document.querySelectorAll('span');
+var firstSpan = document.querySelector('span');
+firstSpan.className = 'cursor';
+
+function matchKeys ({ key }) {
+  for (var j = 0; j < span.length; j++) {
```
Comment (thebearingedge) [inline javascript-typing-tutor/main.js:21]: You don't need a loop for this assignment.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: javascript-typing-tutor/main.js (hunk 3 above)
Comment (yesseniarod) [reply javascript-typing-tutor/main.js]: Screen Shot 2021-03-12 at 11 18 46 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**